docker-compose.yml

```diff
@@ -624,6 +624,37 @@ services:
       - triplestore:virtuoso
     logging: *default-logging
 
+################################################################################
+  # CLB CONSUMERS
+  ################################################################################
+  clb-contact-data-consumer:
+    image: lblod/delta-consumer:0.0.18
```

Comment:
**Suggestion:** Bump to v0.0.24 as we did for the other consumers recently in #401.